playwright-tests/pages/reefKnot.page.ts

```diff
@@ -0,0 +1,59 @@
+import { Page } from 'playwright';
+import { Header } from './components/header';
+import { REEF_KNOT_CONFIG } from '../config';
+import { WalletListModal } from './components/wallet-list-modal';
+import { test } from '@playwright/test';
+import { TIMEOUT } from '../test-data/timeout.data';
+import { StatsBlock } from './components/stats';
+import { WalletModal } from './components/wallet-modal';
+
+export class ReefKnotPage {
+  readonly page: Page;
+  header: Header;
+  walletModal: WalletModal;
+  statsBlock: StatsBlock;
+  walletListModal: WalletListModal;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.header = new Header(this.page);
+    this.walletModal = new WalletModal(this.page);
+    this.statsBlock = new StatsBlock(this.page);
+    this.walletListModal = new WalletListModal(this.page);
+  }
+
+  async goto(param = '') {
+    const link =
+      REEF_KNOT_CONFIG.STAND_LINK_TYPE === 'testnet' ? '/reef-knot' : '';
```

Comment:
could you explain?

Reply:
ReefKnot stand has the `/reef-knot` at the end of the URL (_for localhost stand running it is unnecessary_)
If I save the full stand link on the `baseUrl` - it cuts that part
So we need to put that part in the `goto()` function

Reply:
Thanks for research, fixed